### PR TITLE
Hide some user information via API if user have no enough permission

### PR DESCRIPTION
**Pull request by lunny**
As title.

### CODE REVIEW

File: routers/api/v1/convert/convert.go [hunk 1]
```diff
@@ -237,21 +237,21 @@ func ToTeam(team *models.Team) *api.Team {
 // ToUser convert models.User to api.User
 func ToUser(user *models.User, signed, authed bool) *api.User {
 	result := &api.User{
-		ID:        user.ID,
```
Comment (guillep2k) [inline routers/api/v1/convert/convert.go:240]: What is the purpose of hiding the ID?

**Review by jolheiser — APPROVED** [review]

File: routers/api/v1/convert/convert.go (hunk 1 above)
Comment (guillep2k) [reply routers/api/v1/convert/convert.go:240]: Never mind. 😉

**Review by guillep2k — APPROVED** [review]

**Comment by jolheiser** [comment]
Shoot, forgot to wait for CI before approving. LGTM, just needs some test fixes.

**Comment by guillep2k** [comment]
> 
> 
> Shoot, forgot to wait for CI before approving. LGTM, just needs some test fixes.

Well, the CI is our fourth pair of eyes. :grin:

**Pull request merged**